Source/WebInspectorUI/UserInterface/Views/FindBanner.js

```diff
@@ -227,14 +229,17 @@ WI.FindBanner = class FindBanner extends WI.NavigationItem
 
     // Private
 
-    _inputFieldKeyDown(event)
-    {
-        if (event.keyIdentifier === "Shift")
-            this._searchBackwards = true;
```

Comment:
please remove all mentions of `_searchBackwards` from this file now that it's not used/needed anymore

Reply:
Removed in pr.